bin/src/command/orders.rs

```diff
@@ -757,6 +694,8 @@ impl CommandServer {
                         }
                     };
                 }
+                // shouldn't we use return_success() here, to notify the command server
+                // and the client?
```

Comment:
It could be a great idea :)

Reply:
Yeah, we will need to solve the issue #740 first. I completed the comments.